presto-main/src/main/java/com/facebook/presto/operator/scalar/UrlFunctions.java

```diff
@@ -129,6 +132,42 @@ public static Slice urlExtractParameter(@SqlType(StandardTypes.VARCHAR) Slice ur
         return null;
     }
 
+    @Nullable
+    @Description("encode url")
```

Comment:
This needs a better description, something like "escapes strings so they can be safely included in URL query parameters names and values"